docs/source/simulation_cli.rst

```diff
@@ -144,9 +144,7 @@ indices of the papers you want to start the simulation with.
 
 .. option:: --init_seed INIT_SEED
 
-    Seed for setting the prior indices if the prior_idx option is not used. If
-    the option prior_idx is used with one or more index, this option is
-    ignored.
+    Controls the random selection of papers to train the model on, if :code:`prior_idx` is not specified. The value specified determines the random sequence of prior indices by row numbers. If :code:`prior_idx` is specified, the :code:`--init_seed` argument is ignored.
```

Comment:
`The value specified determines the random sequence of prior indices by row numbers.` This sentence is a bit messy for me. I would personally prefer `The seed value is used to initiate the random selection of relevant and irrelevant records used in the simulation.`

Reply:
Agreed! Thanks, this is much clearer.